javascript-timers/interval.js

```diff
@@ -0,0 +1,18 @@
+const $countDownDisplay = document.querySelector('.countdown-display');
+
+let iD = setInterval(() => {
+  if (iD === 2) {
+    $countDownDisplay.textContent = '3';
+    iD++;
+  } else
+  if (iD === 3) {
+    $countDownDisplay.textContent = '2';
+    iD++;
+  } else
+  if (iD === 4) {
+    $countDownDisplay.textContent = '1';
+    iD++;
+  } else {
+    $countDownDisplay.textContent = '~Earth Beeeelow Us~';
+  }
+}, 1000);
```

Comment:
You shouldn't be manipulating `iD`. `iD` represents the ID of the interval, it doesn't represent the number that we're counting down from. You should be using a separate variable to keep track of the count